plugins/modules/azure_rm_apimanagement_info.py

```diff
@@ -0,0 +1,273 @@
+                + '/resourceGroups' + '/' + self.resource_group \
+                + '/providers' + '/Microsoft.ApiManagement' + '/service' \
+                + '/' + self.service_name + '/apis' + '/' + self.api_id
+
+    def get_url_bytags(self):
+        return '/subscriptions' +'/' + self.subscription_id \
+                + '/resourceGroups' + '/' + self.resource_group \
+                + '/providers' + '/Microsoft.ApiManagement' + '/service' \
+                + '/' + self.service_name + '/apisByTags'
+
+    def get_url_byservice(self):
+        return '/subscriptions' +'/' + self.subscription_id \
+                + '/resourceGroups' + '/' + self.resource_group \
+                + '/providers' + '/Microsoft.ApiManagement' + '/service' \
+                + '/' + self.service_name + '/apis'
+
+    def get_api_info(self):
+        self.url = self.get_url()
+        response = None
+
+        try:
+            response = self.mgmt_client.query(self.url,
+                        'GET',
+                        self.query_parameters,
+                        self.header_parameters,
+                        None,
+                        self.status_code,
+                        600,
+                        30)
+        except CloudError as e:
+            self.log('Could not get the information.{0}'.format(e))
+        try:
+            response = json.loads(response.text)
+        except Exception:
+            response = {'text': response.text}
+
+        return response
+
+    def listbytags(self):
+        self.url = self.get_url_bytags()
+        response = None
+        try:
+            response = self.mgmt_client.query(self.url,
+                        'GET',
+                        self.query_parameters,
+                        self.header_parameters,
+                        None,
+                        self.status_code,
+                        600,
+                        30)
+        except CloudError as e:
+            self.log('Could not get info for the given api tags {0}'.format(e))
+        try:
+          response = json.loads(response.text)
+        except Exception:
+          response = {'text': response.text}
+
+        return response
+
+    def listbyservice(self):
+        self.url = self.get_url_byservice()
+        response = None
+        try:
+            response = self.mgmt_client.query(self.url,
+                        'GET',
+                        self.query_parameters,
+                        self.header_parameters,
+                        None,
+                        self.status_code,
+                        600,
+                        30)
+            response = json.loads(response.text)
+        except CloudError as e:
+            self.log('Could not get info for a given services.{0}'.format(e))
+        try:
+          response = json.loads(response.text)
+        except Exception:
+          response = {'text': response.text}
+
+        return response
+
```

Comment:
expected 2 blanks, found 1

Reply:
Hi, @Fred-sun  I have a small doubt about this feedback. In windows related module if we give any blank lines then shippable will throw an error related to sanity test whereas in azure modules(Python) we are providing extra 2 blank lines. Can you explain to me this rule? Thanks!